.gitignore

```diff
@@ -41,3 +41,5 @@ substrate.code-workspace
 target/
 *.scale
 justfile
+python-venv
+.zed
```

Comment:
This is a bit obscure, maybe add `.zed` file to your global gitignore configuration?
```suggestion
```

Reply:
https://github.com/paritytech/polkadot-sdk/pull/7081/commits/124f5bb404a13e713c6f6183f3be2a4af085b696